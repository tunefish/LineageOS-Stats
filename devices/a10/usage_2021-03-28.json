{
  "10.3": 1,
  "16.0": 66,
  "17.1": 803,
  "18.1": 296,
  "none": 327
}