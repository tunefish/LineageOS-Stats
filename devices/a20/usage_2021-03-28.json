{
  "17.1": 2426,
  "18.1": 159,
  "none": 28
}