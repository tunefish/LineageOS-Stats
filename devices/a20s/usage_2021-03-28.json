{
  "17.1": 2488,
  "none": 1
}