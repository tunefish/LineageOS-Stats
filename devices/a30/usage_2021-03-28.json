{
  "17.1": 34,
  "18.0": 721,
  "18.1": 349,
  "none": 4
}