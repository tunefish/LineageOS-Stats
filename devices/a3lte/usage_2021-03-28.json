{
  "15.1": 688,
  "16.0": 249,
  "17.1": 811
}