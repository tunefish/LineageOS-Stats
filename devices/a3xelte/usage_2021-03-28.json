{
  "14.1": 129,
  "15.1": 141,
  "16.0": 521,
  "17.1": 4446,
  "18.0": 3,
  "18.1": 16,
  "52.8": 1,
  "none": 6
}