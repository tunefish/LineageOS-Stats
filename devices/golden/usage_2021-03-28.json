{
  "13.0": 1219,
  "14.1": 4169,
  "15.1": 5,
  "17.1": 8,
  "none": 1
}